assets/lokomotive-kubernetes/bootkube/resources/charts/bootstrap-secrets/templates/bootstrap-secret.yaml

```diff
@@ -0,0 +1,15 @@
+{{- range $token := .Values.bootstrapTokens }}
+---
+apiVersion: v1
+kind: Secret
+type: bootstrap.kubernetes.io/token
+metadata:
+  # Name MUST be of form "bootstrap-token-<token_id>"
+  name: bootstrap-token-{{ $token.tokenID }}
+  namespace: kube-system
+stringData:
+  description: "Lokomotive generated bootstrap token"
```

Comment:
We may consider adding expiry time here as well.

Reply:
How do we renew that?

Reply:
With `terraform taint`? If it's out of scope, we should remove the cleaner controller and create an issue for it.